__tests__/generate-uuid/generateUuid.test.js

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable no-await-in-loop */
+/* eslint-disable no-plusplus */
+import generateUuid from '../../functions/generate-uuid/1.0';
+
+describe('generateUuid', () => {
+  test('generates a deserialized uuid', async () => {
+    const response = await generateUuid();
+    expect(response.result.length).toBe(36);
+  });
+  test('generates unique values', async () => {
+    const loop = 1000;
```

Comment:
Do you know what the performance impact on the pipelines are? I think 100 would suffice as well

Reply:
100 would suffice indeed.